package.json

```diff
@@ -68,11 +68,12 @@
         "tape": "^5.0.1",
         "ts-jest": "^29.0.5",
         "tsdx": "^0.14.1",
-        "typescript": "^5.0.2"
+        "typescript": "5.5.0-dev.20240603"
     },
     "husky": {
         "hooks": {
             "pre-commit": "pretty-quick --staged"
         }
-    }
```

Comment:
The 5.5.0 is finally out, so we can update it: https://github.com/microsoft/TypeScript/releases